Admin/AbstractAdmin.php

```diff
@@ -3326,4 +3380,17 @@ private function buildRoutes()
             $extension->configureRoutes($this, $this->routes);
         }
     }
+
+    /**
+     * @internal
+     */
+    private function triggerPropertyDeprecation($name)
+    {
+        @trigger_error(sprintf(
+                'Using the "%s::$%s" property is deprecated since version 3.x and will be removed in 4.0.',
+                __CLASS__,
+                $name),
```

Comment:
This is a PSR2 violation, [not fixed by php-cs-fixer yet but I made a PR](https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/2440)